### ISSUE TITLE
SVT play does not work on most programmes. I know why...please, solve it.

### ISSUE THREAD

**Issue by basrieter** [retrospect-addon/plugin.video.retrospect#233]
**[Original report](https://bitbucket.org/basrieter/xbmc-online-tv/issue/233) by Anonymous.**

----------------------------------------

Did you search "All Issues" and not just the "Open Issues" to see if your  
problems was already solved or discussed before? Yes, it´s been discussed, but now I know what the problem is.  
  
  
What channel are you trying to watch? SVT Play  
  
  
What episode are you trying to watch? Many, for example "På spåret", episode 2  
  
  
What is the expected output? What do you see instead? I expect it to flow, but the picture is in slow-motion and the sound is out of sync.  
  
  
**What steps will reproduce the problem?**  
1. Start the "På spåret" via xot-uzg  
**2.**  
**3.**  
  
  
What version of XBMC are you using? On what operating system? A new version (okt 2010) of XBMC on XBOX.  
  
  
Please provide any additional information below. If you start SVT Play on an ordinary computer, it is possible to choose the output quality. When watching SVT Play with xot-uzg, the script uses the highest output quality available (1280x720). XBOX can't play such high quality without problems. SVT Play has changed the maximum quality just a few months ago, so now everyone have trouble with new shows on SVT Play. Before it worked perfectly.  
  
I know I'm right because I've used StreamTransport to rip "På spåret" episode 2 in two different qualities. The 640x360-rip works perfect on the XBOX. But the 1280x720-rip does not - it looks just as i looks when I try the episode via xot-uzg.  
  
To sum this up: Could you please try to fix this problem so that xot-uzg always play the 640x360 version of the SVT Play-shows?  
  
  
**Please ATTACH a complete uzg.log or uzgplugin.log.**  
  
----  
Original item http://code.google.com/p/xot-uzg/issues/detail?id=233
